components/dropbox/sources/common.mjs

```diff
@@ -0,0 +1,40 @@
+import dropbox from "../dropbox.app.mjs";
+
+export default {
+  dedupe: "unique",
+  props: {
```

Comment:
Minor: I prefer to put `dedupe: "unique"` in the individual component files instead of common.mjs. That way it's easier to maintain if we add Dropbox sources in the future that don't use it.